CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # GCP Tamr Configuration module
 
-## v3.1.0 - February 5th 2023
+## v3.0.2 - February 9th 2023
+* Added the following variables to handle custom versions of socket factory for cloudsql
+  * tamr_spark_cloudsql_jar
+  * tamr_cloud_sql_jdbc_url
+
+## v3.0.1 - February 5th 2023
```

Comment:
Look like you changed the version here. Should be 3.1.0 and 3.2.0

Reply:
Ok I just saw the last build after Melissa's PR was pushed to 3.0.1. And that's the latest version, so I assumed it needed to be corrected in our markdown files

<img width="970" alt="Screenshot 2024-02-09 at 12 07 27 PM" src="https://github.com/Datatamer/terraform-gcp-tamr-config/assets/101129946/48ee8a36-dd64-4d43-89d7-6fbdeea1122b">